### DISCUSSION TITLE
Is JIT code cached once it's compiled?

### DISCUSSION THREAD

**Discussion by LouChiSoft** [halide/Halide#8121] [Q&A] [answered]
If I had an program that JIT compiled a pipeline down to Vulkan for example. When I run that application Halide builds the in-memory representation and then once the function is called it compiles that down to code for the machine that is running the application. Is that binary blob of Vulkan code saved to disk/is it possible to save it to disk for subsequent runs.

**Comment by abadams** [comment, marked as answer]
It's cached within the same process, but it isn't stored to disk, so it's not cache across launches of the process.

**Reply by LouChiSoft** [reply]
Dang, that's a shame. Thanks anyway
